r/R/dplyr-funcs-datetime.R

```diff
@@ -618,4 +618,40 @@ register_bindings_datetime_parsers <- function() {
   for (ymd_order in ymd_parser_vec) {
     register_binding(ymd_order, ymd_parser_map_factory(ymd_order))
   }
+
+  register_binding("fast_strptime", function(x,
+                                             format,
+                                             tz = "UTC",
```

Comment:
Could we add a test for non-UTC timezones as well? I don't see any below, I don't think?

Reply:
Good catch. Thanks. Done!